draft-ietf-masque-connect-udp-listen.md

```diff
@@ -188,9 +332,10 @@ validate the target on every datagram and MUST NOT forward individual datagrams
 with unauthorized targets. Proxies can either silently discard such datagrams or
 abort the corresponding request stream.
 
-Unlike non-binding CONNECT-UDP, since the proxy tunnels datagrams from any
-target to clients bound to their respective public IP and ports, the clients
-MUST be ready to handle potential unwanted traffic from unknown destinations.
+Note that if the compression response (COMPRESSION_ASSIGN OR COMPRESSION_CLOSE)
+cannot be immediately sent due to flow or congestion control, an upper limit of
+how much it is willing to proxy SHOULD be set to prevent DDOS-ing. The proxy MAY
```

Comment:
I think you meant "an upper limit on how many compression responses the endpoint is willing to buffer"